src/Mailer/Transport/SimpleQueueTransport.php

```diff
@@ -49,6 +49,12 @@ public function send(Email $email) {
 			'template' => $email->template(), //template() gives 2 values - template and layout
 			'viewVars' => [$email->viewVars()]
 		];
+		
+		foreach ($settings as $setting => $value) {
+			if (empty($value) || array_key_exists(0, $value) && empty($value[0])) {
```

Comment:
it can be simplified to 
```
if (array_key_exists(0, $value) && empty($value[0])) {
```
i dont expect to `'template' => $email->template()` to give null